src/test/java/picard/sam/SamToFastqTest.java

```diff
@@ -290,25 +275,41 @@ public void testOkGroupedFiles(final String samFilename, final String [] groupFi
         }
     }
 
-    private void runWithOutputPerRg(final String samFilename) {
+    private void runWithOutputPerRg(final String samFilename, final boolean outputPerRg) throws IOException {
         final File samFile = new File(TEST_DATA_DIR, samFilename);
-        final String tmpDir = IOUtil.getDefaultTmpDir().getAbsolutePath() + "/";
-        final String[] args = new String[]{
-                "INPUT=" + samFile.getAbsolutePath(),
-                "OUTPUT_PER_RG=true",
-                "OUTPUT_DIR=" + tmpDir,
-        };
-        runPicardCommandLine(args);
+        final List<String> args = new ArrayList<>(Arrays.asList("INPUT=" + samFile.getAbsolutePath()));
```

Comment:
no need for this "optimization" I think that code will be clearer if you split into two lines with List::add.